start-docker.sh

```diff
@@ -5,10 +5,19 @@ cp /app/liquibase.properties /app/data/liquibase.properties
 liquibase --url=jdbc:sqlite:qhana-backend.db --classpath=../liquibase/internal/lib/sqlite-jdbc.jar --driver=org.sqlite.JDBC --changelog-file=../changelog.xml updateTestingRollback
 
 # insert env var for mariadb liquibase config
-eval "echo \"$(cat /app/liquibase.mariadb.properties)\"" > /app/data/liquibase.properties
+cat << EOF > /app/data/liquibase.properties
```

Comment:
Should we write this to another place (outside the docker volume)?

Reply:
Yes, we can run liquibase update in /app